wallet/wallet-cli-lib/src/config.rs

```diff
@@ -58,82 +58,86 @@ impl WalletCliArgs {
 #[derive(Args, Clone, Debug)]
 pub struct CliArgs {
     /// Optional path to the wallet file
-    #[clap(long)]
+    #[clap(long, env = "MINTLAYER_WALLET_FILE")]
     pub wallet_file: Option<PathBuf>,
 
     /// Optional password for a locked wallet
-    #[clap(long)]
+    #[clap(long, env = "MINTLAYER_WALLET_PASSWORD")]
     pub wallet_password: Option<String>,
 
     /// Start staking after the start
-    #[clap(long)]
+    #[clap(long, env = "MINTLAYER_START_STAKING")]
     pub start_staking: bool,
 
     /// Optional RPC address
-    #[clap(long)]
+    #[clap(long, env = "MINTLAYER_NODE_RPC_ADDRESS")]
     pub node_rpc_address: Option<String>,
 
     /// Path to the RPC cookie file. If not set, the value is read from the default cookie file location.
-    #[clap(long)]
+    #[clap(long, env = "MINTLAYER_NODE_RPC_COOKIE_FILE")]
     pub node_rpc_cookie_file: Option<String>,
 
     /// RPC username (either provide a username and password, or use a cookie file. You cannot use both)
-    #[clap(long)]
+    #[clap(long, env = "MINTLAYER_NODE_RPC_USERNAME")]
     pub node_rpc_username: Option<String>,
 
     /// RPC password (either provide a username and password, or use a cookie file. You cannot use both)
-    #[clap(long)]
+    #[clap(long, env = "MINTLAYER_NODE_RPC_PASSWORD")]
     pub node_rpc_password: Option<String>,
```

Comment:
Would it make sense to make these `MINTLAYER_RPC_*`  rather than `MINTLAYER_NODE_RPC_*`? That way, we share the variable between node and wallet, so the wallet connects to the node out of the box if run under the same env variables.

Reply:
I wouldn't care that much about that. One could, in a script or so, assign one to the value of the other. I'd care more about the consistency of naming.

Reply:
> I wouldn't care that much about that. One could, in a script or so, assign one to the value of the other. I'd care more about the consistency of naming.

My thinking here is that with command line parameters, you pass them to each command separately, whereas with env variables, you can assign them once and invoke multiple commands with the same values. Therefore, it is preferable that variables with the same name specify the same thing (e.g. node RPC credentials). As a consequence, the consistency of naming is now a bit more important to achieve across multiple binaries too.

Reply:
I understand, but I'm more afraid of misunderstandings and conflicts than convenience. I'd rather have people get something not working because they didn't configure it right than not working due to some side effect we haven't thought through.